### DISCUSSION TITLE
NdisApiDll Does Not Exist

### DISCUSSION THREAD

**Discussion by toddalbers** [HavenDV/H.Vpn#15] [General]
Apologies for this being a very basic question.  But, I downloaded the solution.  I changed the solutions startup project to be the NdisApiDemo project.  I then rebuilt the entire solution successfully.  I ran the solution which ran  NdisApiDemo project.  It errors on the following lines of code in NdisApi.cs:
`if (!NdisApiDllExists())
    throw new DllNotFoundException("Missing NDIS DLL.");`

I believe it is trying to load: ndisrd.sys
But, I could be wrong.
Any suggestions?

**Comment by HavenDV** [comment]
Hello. I'm not sure I understand you. There are no such lines or Ndis loading in the code of this project
